config/routes.rb

```diff
@@ -23,6 +23,11 @@
         post "passwords/check_otp", to: "passwords#check_otp"
         post "passwords/reset", to: "passwords#reset_password"
       end
+
+      post "classroom/teacher", to: "classrooms#create"
+      get "classrooms", to: "classrooms#get_classrooms"
+      put "classroom/:id", to: "classrooms#update_classroom"
+      delete "classroom/:id", to: "classrooms#delete_classroom"
```

Comment:
Let's follow REST API principles 


```suggestion
      put "classrooms/:id", to: "classrooms#update_classroom"
      delete "classrooms/:id", to: "classrooms#delete_classroom"
```